reseller-store.php

```diff
@@ -40,7 +40,7 @@
 	 *
 	 * @var string
 	 */
-	const VERSION = '1.0.0';
+	const VERSION = '0.2.0';
```

Comment:
1.2.0

Reply:
there is no such thing as version 1.2.0 see releases https://github.com/godaddy/wp-reseller-store/releases